import_export/resources.py

```diff
@@ -852,16 +841,29 @@ def import_data_inner(
             result.add_dataset_headers(dataset.headers)
 
         for i, row in enumerate(dataset.dict, 1):
-            with atomic_if_using_transaction(using_transactions, using=db_connection):
+            with atomic_if_using_transaction(using_transactions and not self._meta.use_bulk, using=db_connection):
```

Comment:
Could you add a test which proves that the value of `self._meta.use_bulk` gets passed to `atomic_if_using_transaction()` at this point?